ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -277,7 +277,11 @@ const ConfirmPageContainer = (props) => {
                 : onSubmit
             }
             submitText={t('confirm')}
-            submitButtonType={isSetApproveForAll ? 'danger-primary' : 'primary'}
+            submitButtonType={
+              isSetApproveForAll && isApprovalOrRejection
```

Comment:
Note: `isApprovalOrRejection` is `true` when it is an approval transaction, and `false` when it's a revocation. The naming of the variable seems to imply it is true in both cases. It could likely be named better.